pkg/k8s/scanner/io.go

```diff
@@ -15,29 +16,56 @@ import (
 
 var r = regexp.MustCompile("\\\\|/|:|\\*|\\?|<|>")
 
-func createTempFile(artifact *artifacts.Artifact) (string, error) {
+func generateTempFileByArtifact(artifact *artifacts.Artifact, tempFolder string) (string, error) {
```

Comment:
Just a thought, would they be also cleaned when we run `trivy clean`?

Reply:
these temporary files remove after the k8s scan.
so I don't think we should use `trivy clean` here

Reply:
Trivy creates these files inside temporary folder, so I think OS can take care of it.
or do i miss something? thanks

Reply:
Fair enough 👍